pkg/pxc/service.go

```diff
@@ -144,6 +145,13 @@ func NewServiceProxySQLUnready(cr *api.PerconaXtraDBCluster) *corev1.Service {
 	return obj
 }
 
+func NewServiceProxySQLUnreadyName(cr *api.PerconaXtraDBCluster) types.NamespacedName {
```

Comment:
Consider rename to ProxySQLUnreadyServiceNamespacedName

Reply:
fbb29694